sample_app/sampleapp_common.c

```diff
@@ -422,6 +458,12 @@ static int app_release_ind (
          p_result->pnio_status.error_code_2);
    }
 
+   if (p_appdata->arguments.verbosity > 0)
+   {
+      printf ("Setting outputs to default values (due to RELEASE)\n");
```

Comment:
Remove if printout is moved to app_set_outputs_default_value()

Reply:
Done